dotcms-integration/src/test/java/com/dotcms/graphql/datafetcher/page/ContentMapDataFetcherTest.java

```diff
@@ -121,4 +135,130 @@ public void testRawFieldDoesNotAddMissingBaseField() throws Exception {
         assertEquals(rawJson, map.get("customField_raw"));
         assertFalse("Parsed base field should not be added", map.containsKey("customField"));
     }
+
+    /**
+     * Verifies that when no variant is set in the request,
```

Comment:
are these tests ITs? it seems they are unit tests as many objects are being mocked.